ui/src/components/Select.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+import CreatableSelect from 'react-select/creatable'
+import { KeyOption, keyOptions } from 'components/SelectData'
+import { ActionMeta, OnChangeValue } from 'react-select'
+
+export default class Select extends Component<{}> {
+  handleChange = (newValue: OnChangeValue<KeyOption, true>, actionMeta: ActionMeta<KeyOption>) => {
+    console.group('Value Changed')
+    console.log(newValue)
+    console.log(`action: ${actionMeta.action}`)
+    console.groupEnd()

```

Comment:
we can kill all these logs. Also sidenote, I never knew console.groupEnd() was a thing. Neat.